core/templates/css/oppia-material.css

```
@@ -2205,6 +2205,9 @@ th.mat-header-cell, td.mat-cell, td.mat-footer-cell {
 .mat-radio-button.mat-radio-disabled .mat-radio-label-content {
   color: rgba(0, 0, 0, 0.38); }
 
+.mat-radio-button .mat-radio-label {
+  align-items: flex-start; }
+
```

```suggestion
.mat-radio-button .mat-radio-label {
  align-items: flex-start; 
}

```

Also if possible can we move it to the component? maybe using /deep/ will help us get the required effect. As mat-radio-button and label are used at number of places and this might break there  display.

---

> Also if possible can we move it to the local html of the component? maybe using /deep/ will help us get the required effect. As mat-radio-button and label are used at number of places and this might break their display.

Hey @Rijuta-s , moved the css to local html component and also made changes to ensure no cluttering of difficulty levels. Please have a look.

---

Thanks @paprajapati9 .